packages/cli-ui/src/containers/ProjectContainer/ProjectContainer.tsx

```diff
@@ -1,66 +1,62 @@
-import React, { useMemo, useState, useEffect } from 'react'
+import React, { useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+
 import { CheckBoxTheme, Footer } from 'components'
-import {NavLink} from 'react-router-dom'
+import { NavLink } from 'react-router-dom'
 import useProjectContainer from './projectContainer.hook'
 import { Routes } from 'router'
+
 import logo from '../../../public/logo192.png'
 import css from './style.module.css'
 
 export default function ProjectContainer () {
-  const { tabs, activeTab, handleSetTab } = useProjectContainer()
-  const [value, setValue] = useState(JSON.parse(localStorage.getItem('thememode')))
-
-  useEffect(() => {
-    setValue(JSON.parse(localStorage.getItem('thememode')))
-  }, [value])
-
-  const actived = value ? 'actived' : ''
+  const { t } = useTranslation('project')
+  const { locale, tabs, activeTab, isDarkTheme } = useProjectContainer()
```

Comment:
@VladKalachev @goldLifter  Тут вот под вопросом стоит ли вынести использование и импортирование данного хука в кастомный хук и экспортировать уже из него, чтобы изолировать всю логику и сделать компонентик максимально тупым?)
```suggestion
  const { t, locale, tabs, activeTab, isDarkTheme } = useProjectContainer()
```

Reply:
с точки зрения чистоты можно, но может быть не очевидно для других такое решение. склоняюсь оставить как есть с отдельным const { t } = useTranslation('project')